spec/lib/mcb/commands/courses/edit_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'mcb_helper'
+
+describe 'mcb courses edit' do
+  def edit(provider_code, course_codes, *input_cmds)
+    stderr = nil
+    output = with_stubbed_stdout(stdin: input_cmds.join("\n"), stderr: stderr) do
+      cmd.run([provider_code] + course_codes)
+    end
+    [output, stderr]
```

Comment:
Looking at this, it feels like `with_stubbed_stdout` should return `(stdin, stdout)`.

Reply:
I've stolen this pattern from a few of the other `mcb` specs, we should fix this across the piece on a separate PR.